### PR TITLE
Eliminate callbacks from client_events.js

### PR DESCRIPTION
**Pull request by pianiststickman**

### CODE REVIEW

File: commands/admin/addtime.js [hunk 1]
```diff
@@ -19,22 +19,16 @@ module.exports.load = (client) => {
...
-              setTimeout(() => {
-                m.delete()
-              }, client.settings.res_destruct_time * 1000)
-            })
-        })
-      })
+      userInfo.current_session_playtime += delta
+      userInfo.overall_session_playtime += delta
+      await client.writeUserData(args[0].replace(/[<@!>]/g, ''), userInfo)
+      let m = await message.reply('added time to user.')
```
Comment (sit) [inline commands/admin/addtime.js]: We could maintain the old code path which is "faster" to return from the command. This seems like a legit use of `.then`. Also, neither the original nor this re-factor includes proper error handling :-)

File: commands/general/stats.js [hunk 2]
```diff
@@ -4,7 +4,7 @@ const moment = require('moment')
 
 module.exports.load = (client) => {
   client.commands['stats'] = {
-    run (message) {
+    async run (message) {
```
Comment (sit) [inline commands/general/stats.js]: See #35 also.

Comment (pianiststickman) [reply commands/general/stats.js]: Yep, probably gonna make a few changes to that one and check it in as well. Just want to un-callback-ify as much of the code as possible before I continue on the replat.

File: commands/admin/addtime.js (hunk 1 above)
Comment (pianiststickman) [reply commands/admin/addtime.js]: I think the whole self-destructing message thing should probably be a library method anyway. We use that technique all over the place and we could put the error-handling there.

**Pull request merged**